doc/scripts/images.dvc

```diff
@@ -1,5 +1,5 @@
 outs:
-- md5: e0c1334651a2b06141769144677fdff5.dir
-  size: 33377667
-  nfiles: 204
+- md5: 453d9a7563e7c2ec20cc0ed14716053c.dir
```

Comment:
The changes doesn't make sense to me. I expect to see `nfiles` in `doc/scripts/images.dvc` increase by 1, and `nfiles` in `test/baseline/test.dvc` decrease by 1.

Reply:
At the top, Paul said that he removed one test.